<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeDialog</class>
 <widget class="QDialog" name="HomeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>435</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DownZemAll - Getting Started</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_1">
     <property name="leftMargin">
      <number>48</number>
     </property>
     <property name="topMargin">
      <number>32</number>
     </property>
     <property name="rightMargin">
      <number>48</number>
     </property>
     <property name="bottomMargin">
      <number>32</number>
     </property>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Getting Started</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="subTitleLabel">
       <property name="text">
        <string>Choose which category of document to download.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>360</width>
         <height>32</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="buttonContent">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Web Page Content</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/menu/wizard_add_32x32.png</normaloff>:/icons/menu/wizard_add_32x32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="description">
        <string>Links and media in a HTML page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="buttonBatch">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Batch of Files (with regular expression)</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/menu/tool_add_26x26.png</normaloff>:/icons/menu/tool_add_26x26.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="description">
        <string>Single file, batch of files, regular expression link</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="buttonStream">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Video/Audio Stream</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/menu/youtube_48x48.png</normaloff>:/icons/menu/youtube_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="description">
        <string>Stream from Youtube and other video stream sites</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="buttonTorrent">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Magnet Link, Torrent</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/menu/preference_torrent_32x32.png</normaloff>:/icons/menu/preference_torrent_32x32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="description">
        <string>Use bittorrent protocol to download a .torrent file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBox">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonContent</tabstop>
  <tabstop>buttonBatch</tabstop>
  <tabstop>buttonStream</tabstop>
  <tabstop>buttonTorrent</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>acceptPaused()</slot>
 </slots>
</ui>
